cms/templates/partials/nhs_components/pagination.html

```diff
@@ -1,27 +1,38 @@
-{# page pagination template page,paremeters, prev_page: the previous page number, next_page: the next page, total: total page count #}
-{#TODO: need an if statement to hide prev at the start and also at the end dont show next when equals total#}
+{# page pagination template page #}
+{#  parameters: #}
+{#  * prev_page: the previous page number #}
+{#  * next_page: the next page #}
+{#  * total: total page count #}
 
 <nav class="nhsuk-pagination" role="navigation" aria-label="Pagination">
   <ul class="nhsuk-list nhsuk-pagination__list">
-    <li class="nhsuk-pagination-item--previous">
-      <a class="nhsuk-pagination__link nhsuk-pagination__link--prev" href="/section/treatments">
-        <span class="nhsuk-pagination__title">Previous</span>
-        <span class="nhsuk-u-visually-hidden">:</span>
-        <span class="nhsuk-pagination__page">{{ prev_page }} of {{ total }}</span>
-        <svg class="nhsuk-icon nhsuk-icon__arrow-left" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" width="34" height="34">
-          <path d="M4.1 12.3l2.7 3c.2.2.5.2.7 0 .1-.1.1-.2.1-.3v-2h11c.6 0 1-.4 1-1s-.4-1-1-1h-11V9c0-.2-.1-.4-.3-.5h-.2c-.1 0-.3.1-.4.2l-2.7 3c0 .2 0 .4.1.6z"></path>
-        </svg>
-      </a>
-    </li>
-    <li class="nhsuk-pagination-item--next">
-      <a class="nhsuk-pagination__link nhsuk-pagination__link--next" href="/section/symptoms">
-        <span class="nhsuk-pagination__title">Next</span>
-        <span class="nhsuk-u-visually-hidden">:</span>
-        <span class="nhsuk-pagination__page">{{ next_page }} of {{ total }}</span>
-        <svg class="nhsuk-icon nhsuk-icon__arrow-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" width="34" height="34">
-          <path d="M19.6 11.66l-2.73-3A.51.51 0 0 0 16 9v2H5a1 1 0 0 0 0 2h11v2a.5.5 0 0 0 .32.46.39.39 0 0 0 .18 0 .52.52 0 0 0 .37-.16l2.73-3a.5.5 0 0 0 0-.64z"></path>
-        </svg>
-      </a>
-    </li>
+    {% if prev_page and prev_page != 1 %}
```

Comment:
Surprised that we want to hide this if the previous page is page 1; surely that means we're on page 2 and there's a page we can turn back to? I'm probably misunderstanding the pagination since the tests look like they'd catch that case. [non-blocking]

Reply:
Not sure why that is but it works. The numbers we get from the search pages work with this. :)

I'll have a look.